hypothesis-python/tests/ghostwriter/recorded/hypothesis_module_magic.txt

```diff
@@ -50,7 +48,7 @@ def test_fuzz_note(value: str) -> None:
     hypothesis.note(value=value)
 
 
-@given(r=st.nothing())
+@given(r=st.from_type(random.Random))
```

Comment:
It seems like it's used as a defered strategy because of the `hypothesis.errors.SmallSearchSpaceWarning`. I've added the randoms to the `_global_type_lookup`.